### PR TITLE
added $BLOX

### PR DESCRIPTION
**Pull request by ArthurHoeke**
Added $BLOX distributor

Token contract:
https://viewblock.io/zilliqa/address/zil1gf5vxndx44q6fn025fwdaajnrmgvngdzel0jzp

Distributor contract:
https://viewblock.io/zilliqa/address/zil1v3zepjnau579yadpk4szpx0n6ew6gdplah27v0

### CODE REVIEW

File: config/config.yml [hunk 1]
```diff
@@ -58,6 +58,24 @@ mainnet:
...
+      reward_token_address_hex: "0x4268C34dA6Ad41a4cDeAa25cdEF6531Ed0c9a1A2"
+      distributor_name: BLOX
+      distributor_address_hex: "0x644590ca7de53c5275a1b5602099f3d65da4343f"
+      developer_address: zil1gy9v2fy4nsa5n5mx7grtt5sn205d6qduxkl84t
+      emission_info:
+        epoch_period: 604800 # 1 week
+        tokens_per_epoch: "1086900"
+        tokens_for_retroactive_distribution: "0"
+        retroactive_distribution_cutoff_time: 0
+        distribution_start_time: 1630483200
```
Comment (RavenXce) [inline config/config.yml:72]: This start time is 1 month ago, meaning 4 epochs will be generated?

Comment (ArthurHoeke) [reply config/config.yml]: Ah that's not right, I've updated it to 1632902400

Comment (RavenXce) [reply config/config.yml]: Sorry, could you resolve the merge conflicts?

Comment (ArthurHoeke) [reply config/config.yml]: For easy I've closed this pull request and created a fresh one over at
https://github.com/Switcheo/zap-api/pull/21

**Pull request closed without merging**